keyboards/owlab/suit80/ansi/info.json

```
@@ -0,0 +1,100 @@
+{
+    "keyboard_name": "suit80",
+    "url": "https://www.qwertykeys.com/products/jrrgvdsyu47wer",
```

```suggestion
    "url": "",
```
The specified URL points to the Jelly Epoch 75% page; it should be either removed or replaced with a proper URL.